src/org/usfirst/frc/team467/robot/DriverStation.java

```diff
@@ -15,7 +15,11 @@
 	// Mapping of functions to Controller Buttons for normal operation
 	// TODO: Create enum for buttons
 	
+	double maxAcceleration = 0.1;
```

Comment:
This maxAcceleration value should go in the RobotMap class. It could vary depending on the robot.